### PR TITLE
Feature/events v1 without record

### PR DESCRIPTION
**Pull request by robYP**
create event endpoints: create/update/delete with JWT authentication

### CODE REVIEW

File: repository/event.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+def delete_event_by_id(db:Session, event_id:int):
+    db_event = db.query(Event).filter(Event.id == event_id).first()
+    if not db_event:
+        return None
+    db.delete(db_event)
+    db.commit()
+    return db_event
+
+
+def delete_post_by_id(db: Session, post_id: int):
```
Comment (jonyig) [inline repository/event.py]: 這個用意為何？

File: repository/event.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    db_post = db.query(Post).filter(Post.id==post_id).first()
+    if not db_post:
+        return None
+    db.delete(db_post)
+    db.commit()
+    return db_post
+
+def update_event_by_id(user: User, event_id: int, event: EventCreate, db: Session):
+    db_event = db.query(Event).filter(Event.id == event_id).first()
```
Comment (jonyig) [inline repository/event.py]: 這邊跟 21 行，有相同的 code ，要不要嘗試合併？

**Comment by robYP** [comment]
curl for LIST/CREATE/UPDATE:

**List event:**
curl -X 'GET' \
  'http://127.0.0.1:8000/v1/events/?skip=0&limit=100' \
  -H 'accept: application/json'

**Create event:**
curl -X 'POST' \
  'http://127.0.0.1:8000/v1/events/' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJyb2JjaGVuMTIiLCJpZCI6NiwiZXhwIjoxNzI0MDkxMTI3fQ.ZSEeOu5C6brQfzVuN84w6azigPnQDYCFql2XfumZRW0' \
  -H 'Content-Type: application/json' \
  -d '{
  "name": "ORDER_COMPLETED",
  "route": "LINE"
}'

**Update event:**
curl -X 'PUT' \
  'http://127.0.0.1:8000/v1/events/12' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJyb2JjaGVuMTIiLCJpZCI6NiwiZXhwIjoxNzI0MDkxMTI3fQ.ZSEeOu5C6brQfzVuN84w6azigPnQDYCFql2XfumZRW0' \
  -H 'Content-Type: application/json' \
  -d '{
  "name": "ORDER_SHIPPED",
  "route": "SMS"
}'

File: repository/event.py (hunk 1 above)
Comment (robYP) [reply repository/event.py]: 誤放的程式碼

File: repository/event.py (hunk 2 above)
Comment (robYP) [reply repository/event.py]: 收到，新版已合併

File: repository/event.py (hunk 1 above)
Comment (robYP) [reply repository/event.py]: 已更改在 commit e586dba3673fe0d2991198b232868ea8911ccd78

File: repository/event.py (hunk 2 above)
Comment (robYP) [reply repository/event.py]: 已更改在 commit
e586dba3673fe0d2991198b232868ea8911ccd78

**Review by jonyig — APPROVED** [review]

**Pull request merged**